website/server/controllers/api-v3/groups.js

```diff
@@ -329,7 +329,6 @@ api.rejectGroupInvite = {
  * @apiGroup Group
  *
  * @apiParam {string} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
- * @apiParam {string="remove-all","keep-all"} keep Query parameter - Whether to keep or not challenges' tasks. Defaults to keep-all
```

Comment:
Removing this behavior is a breaking change. So we can't do it. I think it would be better to correct this behavior and have `keep-all` keep challenges _and_ tasks, while `remove-all` should remove challenges _and_tasks. Does that make sense?